### PR TITLE
Check species eligibility #86

### PR DESCRIPTION
**Pull request by ami-day**
Added function to check eligibility of species ontology labels in the dataset. Checks the following:

- Only Human or Mouse species are included in the dataset. If an alternative species is included, an Assertion error is raised: "1 or more species are not eligible. Species must be either Homo sapiens or Mus musculus. Please remove the relevant biomaterials from the dataset and run again."

- If both Human and Mouse species are included in the dataset, an assertion error is raised: "AssertionError: Only 1 species is allowed per SCEA E-HCAD id. Please split the dataset by species and run them separately."

- If any biomaterials contain pooled species (i.e. pooled samples from human and mouse), an assertion error is raised: ""The dataset contains biomaterials linked to >1 species (pooled). To be eligible for SCEA, each biomaterial must be linked to 1 species only (Human or Mouse). Please remove the relevant biomaterials from the dataset and run again."

### CODE REVIEW

File: hca-to-scea/test/golden/arguments.csv [hunk 1]
```diff
@@ -1,12 +1,12 @@
 spreadsheet,HCA project uuid,study accession,E-HCAD accession,curator initials,experiment type,factor values,public release date,hca last update date
-SRP151260.xlsx,c893cb57-5c9f-4f26-9312-21b85be84313,SRP151260,50,AD,differential,disease,2009-02-27,2021-04-29
-SRP274475.xlsx,07073c12-8006-4710-a00b-23abdb814904,SRP274475,36,AD,differential,organ,2021-09-30,2020-11-25
-ERP121676_1.xlsx,955dfc2c-a8c6-4d04-aa4d-907610545d11,ERP121676,100,AD,baseline,individual,2021-09-30,2020-11-25
-ERP121676_2.xlsx,955dfc2c-a8c6-4d04-aa4d-907610545d11,ERP121676,100,AD,baseline,individual,2021-09-30,2020-11-25
-ERP121676_3.xlsx,955dfc2c-a8c6-4d04-aa4d-907610545d11,ERP121676,100,AD,baseline,individual,2021-09-30,2020-11-25
+#SRP151260.xlsx,c893cb57-5c9f-4f26-9312-21b85be84313,SRP151260,50,AD,differential,disease,2009-02-27,2021-04-29
```
Comment (amnonkhen) [inline hca-to-scea/test/golden/arguments.csv]: Why is this commented out?

**Review by amnonkhen — CHANGES_REQUESTED** [review]

File: hca-to-scea/test/golden/arguments.csv (hunk 1 above)
Comment (ami-day) [reply hca-to-scea/test/golden/arguments.csv]: My plan is to keep these commented out in the pull requests specific to certain test cases, and then once the multiple pull requests have been merged into master, to create a new branch from master where I test these positive tests.

Comment (ami-day) [reply hca-to-scea/test/golden/arguments.csv]: (the other option would be to delete those lines instead of commenting them out)

Comment (amnonkhen) [reply hca-to-scea/test/golden/arguments.csv]: We don't keep commented out scenarios in different branches.
We either make the scenario work or delete it.
When we merge each branch to the common branch it should include the union of all test cases.
This is what would protect us against breaking existing functionality when we change the code in the future.

Comment (ami-day) [reply hca-to-scea/test/golden/arguments.csv]: Ok, I'll delete the commented out part

**Review by amnonkhen — DISMISSED** [review]

**Review by amnonkhen — APPROVED** [review]

**Pull request merged**